### PR TITLE
Project review

### PR DESCRIPTION
**Pull request by yosaddis**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+				<label for="open-source"> Open Source Challenge</label><br>
+				<input type="checkbox" value="Video editing" />
+				<label for="video-editing"> Video Editing</label><br>
+				<input type="checkbox" name="ui-ux" value="UI/UX project" />
+				<label for="ui-ux"> UI/UX Project</label>
+			</div>
+			<br />
+			<!--<textarea placeholder="Add any ectra comments" cols="10">
+				
+			</textarea><br>-->
```
Comment (yosaddis) [inline index.html:55]: Please don't leave unused code as a comment in your code base. Unused code as a comment makes your code unclean.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+	<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+	<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+	<title>Survey Form</title>
+	<link rel="stylesheet" href="./style.css" />
+</head>
+
+<body>
+	<div class="container">
+		<h1 id="title">Survey</h1>
+		<p id="description">
```
Comment (yosaddis) [inline index.html:15]: Kindly change your form name into more descriptive one. So that the users can understand what is the survey is about.

**Review by yosaddis — COMMENTED** [review]
Changes required ♻️ 

- Please use a more descriptive Readme file. As a means to others can understand what your project is about.
- Kindly add linter files to your project to because it will guide you to have a good HTML & CSS practices.
- Please use more descriptive commit messages. So that the code reviewers can understand what is changed on this commit. And what are you trying to change. 

Submit for a re-review when you are done. 😀

**Review by Olamarx — APPROVED** [review]
Hi @yosaddis,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Inline comment ✔️ 
- Polite comment ✔️ 
- Comment with reason ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**